src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -52,14 +52,14 @@ By default, our [Helm Chart](/docs/kubernetes-pixie/kubernetes-integration/insta
 
 #### `hostNetwork` and `privileged` [#hostnetwork-privileged]
 
-Most users and Kubernetes distributions configure the control plane metrics endpoints to listen only in the loopback interface, i.e. `localhost`. For this reason, the control plane component is deployed with `hostNetwork: true` by default when `privileged` is set to `true` (the default).
+In versions lower than v3, when the integration is deployed using `privileged: false`, the `hostNetwork` setting for the control plane component will be also be set to `false`.
 
-When the integration is deployed using `privileged: false`, the `hostNetwork` setting for the control plane component will be also be set to `false`. We chose to do it this way because otherwise, we would not be honoring the intent users have when they set `privileged: false`. Unfortunately, deploying without `hostNetwork` will cause control plane scraping to fail in most environments, which will result in missing metrics or the `nrk8s-controlplane` pods getting stuck into a `CrashLoopBackoff` state. This is a limitation of Kubernetes itself, as control plane cannot be monitored without `hostNetwork` unless components are manually configured to do so.
-
-As it is a common setting to deploy the integration in unprivileged mode (`privileged: false`), but still consider acceptable running the control plane pods with `hostNetwork`. This can be achieved by setting `controlPlane.unprivilegedHostNetwork` to `true`: This will tell the chart to deploy the control plane component with `hostNetwork: true`, despite the value of the higher-level `privileged` flag.
+With the latest version, `privileged` only affects node pods adding privileged to the security context so we can get more metrics from the host. All the components of this integration are set up as `hostNetwork: false` by default, except the pods that get metrics from the control plane. This is because most users and Kubernetes distributions configure the control plane metrics endpoints to only listen in the loopback interface, i.e. `localhost`.
```

Comment:
```suggestion
In version 3 and newer, the `privileged` flag affects only the `securityContext` objects, that is, whether containers are run as root with access to host metrics or not. All the integration components default now to `hostNetwork: false`, except the pods that get metrics from the control plane that have `hostNetwork: true` as it is required to reach the control plane endpoints in most distributions. Both of these settings can be changed, individually or globally, using _______.
```

Reply:
Solved in [#c326625](https://github.com/newrelic/docs-website/pull/7322/commits/c32662541d45e88c090fd3eeecea3935bdd6e660)